modules/loader-utils/src/lib/node/fs.ts

```diff
@@ -4,74 +4,37 @@ import {toArrayBuffer} from './buffer';
 import {promisify2, promisify3} from './promisify';
 
 export type {Stats, ReadStream, WriteStream} from 'fs';
-
-export let readdir;
 /** Wrapper for Node.js fs method */
-export let stat;
-export let statSync: typeof fs.statSync;
+export const readdir: any = promisify2(fs.readdir);
+/** Wrapper for Node.js fs method */
+export const stat: any = promisify2(fs.stat);
+export const statSync = fs.statSync;
 
 /** Wrapper for Node.js fs method */
-export let readFile;
+export const readFile: any = fs.readFile;
 /** Wrapper for Node.js fs method */
-export let readFileSync: typeof fs.readFileSync;
+export const readFileSync = fs.readFileSync;
 /** Wrapper for Node.js fs method */
-export let writeFile;
+export const writeFile: any = promisify3(fs.writeFile);
 /** Wrapper for Node.js fs method */
-export let writeFileSync: typeof fs.writeFileSync;
+export const writeFileSync = fs.writeFileSync;
 
 // file descriptors
 
 /** Wrapper for Node.js fs method */
-export let open;
+export const open: any = fs.open;
 /** Wrapper for Node.js fs method */
-export let close: (fd: number) => Promise<void>;
+export const close = (fd: number) =>
+  new Promise<void>((resolve, reject) => fs.close(fd, (err) => (err ? reject(err) : resolve())));
 /** Wrapper for Node.js fs method */
-export let read;
+export const read: any = fs.read;
```

Comment:
Forcing type to `any` to disable TS errors.

I do not think these functions are being used correctly, but I don't know enough about the system to fix them.